fxdc/config.py

```
@@ -1,33 +1,50 @@
+from json import load
```

Ruff (0.12.2)

`json.load` imported but unused

Remove unused import: `json.load`

(F401)